### PR TITLE
chore(service-account): use error codes from SDK

### PR DESCRIPTION
**Pull request by rkpattnaik780**
Error handling for service-account operations.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [X] SDK Integration

### CODE REVIEW

File: pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package errorutils
```
Comment (rkpattnaik780) [inline pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go:1]: @wtrocki should we move this file to SDK, lik the other SDKs have?

File: pkg/cmd/serviceaccount/create/create.go [hunk 2]
```diff
@@ -140,8 +141,13 @@ func runCreate(opts *options) error {
 		defer httpRes.Body.Close()
 	}
 
-	if err != nil {
-		return err
+	if apiErr := errorutils.GetAPIError(err); apiErr != nil {
+		switch apiErr.GetError() {
+		case "service_account_limit_exceeded":
```
Comment (wtrocki) [inline pkg/cmd/serviceaccount/create/create.go:146]: I think we have Enums defined in SDK for those

File: pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go (hunk 1 above)
Comment (wtrocki) [reply pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go:1]: +1 on adding this to SDK. Thank you

File: pkg/cmd/serviceaccount/create/create.go (hunk 2 above)
Comment (rkpattnaik780) [reply pkg/cmd/serviceaccount/create/create.go:146]: I tried that first. The enums are defined in a different manner in the openAPI [file](https://github.com/redhat-developer/app-services-sdk-go/blob/main/.openapi/service-accounts.yaml#L346), more like accepted options.

File: pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go (hunk 1 above)
Comment (rkpattnaik780) [reply pkg/cmd/serviceaccount/svcaccountcmdutil/errorutils/error_sdk.go:1]: Raised PR https://github.com/redhat-developer/app-services-sdk-go/pull/555.

File: pkg/core/localize/locales/en/cmd/serviceaccount.en.toml [hunk 3]
```diff
@@ -43,6 +43,9 @@ one = 'internal server error'
 one = 'you are forbidden to {{.Operation}} this service account'
 other = 'you are forbidden to {{.Operation}} these service accounts'
 
+[serviceAccount.common.error.limitExceeded]
+one = 'account can not have more than 50 service accounts'
```
Comment (wtrocki) [inline pkg/core/localize/locales/en/cmd/serviceaccount.en.toml:47]: We cannot put any backend driven business logic into message. 
50 account limit is just config on backend that can change etc.

```suggestion
one = 'account can not have more than 50 service accounts'
```

**Review by wtrocki — APPROVED** [review]

**Review by wtrocki — APPROVED** [review]
As discussed. This is good to be merged

**Pull request merged**